libs/spark/src/Unstable_Tag/Unstable_Tag.tsx

```diff
@@ -0,0 +1,285 @@
+                backgroundColor: darken(backgroundColor, 0.4),
+              }),
+              ...(props.variant === 'subtle' && {
+                backgroundColor: darken(backgroundColor, 0.2),
+              }),
+            },
+          },
+          '&.Mui-focusVisible, &:focus-visible': {
+            boxShadow: `0px 0px 2px 4px ${theme.unstable_palette.teal[300]}`,
+          },
+        }),
+        ...(props.onDelete && {
+          // artificially increase specificity to override MUI
+          '&&': {
+            '&:hover': { backgroundColor },
+            '&:active': { backgroundColor },
+          },
+          '&.Mui-focusVisible, &:focus-visible': {
+            boxShadow: `0px 0px 2px 4px ${theme.unstable_palette.teal[300]}`,
+          },
+        }),
+      };
+    },
+    label: (props: Unstable_TagProps) => ({
+      color: 'inherit',
+      padding: 0, // unset MUI
+      ...(props.size === 'medium' && tagFontVariantMedium),
+      ...(props.size === 'small' && tagFontVariantSmall),
+      // artificially shift label down by 1px to appear more centered; especially apparent when beside an icon
+      marginTop: 1,
+      marginBottom: -1,
+    }),
+    deleteIcon: (props: Unstable_TagProps) => ({
+      borderRadius: 2,
+      color: 'inherit',
+      height: '1em',
+      margin: '0 0 0 4px',
+      width: '1em',
+      ...(props.size === 'medium' && {
+        fontSize: theme.typography.pxToRem(16),
+      }),
+      ...(props.size === 'small' && {
+        fontSize: theme.typography.pxToRem(12),
+      }),
+      '&:hover': {
+        color: 'inherit', // override MUI
+        ...(props.variant === 'bold' && {
+          backgroundColor: alpha(theme.unstable_palette.neutral[600], 0.2),
+        }),
+        ...(props.variant === 'subtle' && {
+          backgroundColor: alpha(theme.unstable_palette.neutral[600], 0.1),
+        }),
+      },
+      '&:active': {
+        ...(props.variant === 'bold' && {
+          backgroundColor: alpha(theme.unstable_palette.neutral[600], 0.3),
+        }),
+        ...(props.variant === 'subtle' && {
+          backgroundColor: alpha(theme.unstable_palette.neutral[600], 0.2),
+        }),
+      },
+    }),
+    icon: (props: Unstable_TagProps) => ({
+      color: 'inherit',
+      height: '1em',
+      margin: '0 4px 0 0',
+      width: '1em',
+      ...(props.size === 'medium' && {
+        fontSize: theme.typography.pxToRem(16),
+      }),
+      ...(props.size === 'small' && {
+        fontSize: theme.typography.pxToRem(12),
+      }),
+    }),
+    avatar: {
+      // blocked until `Unstable_Avatar` initial implementation
+    },
+  }),
+  { name: 'MuiSparkUnstable_Tag' }
+);
+
+const Unstable_Tag: OverridableComponent<Unstable_TagTypeMap> = React.forwardRef(
+  function Unstable_Tag(props, ref) {
+    const {
+      classes: classesProp,
+      color = 'neutral',
+      onClick,
+      onDelete,
+      size = 'medium',
+      variant = 'bold',
```

Comment:
Should the controls default to these values as well?

Reply:
I don't want them to be. It'd be nice for the documentation, but the point of our "default" story is to validate the styling of when no props are specified. and having the controls explicitly default to that would cause them to not default, but instead take on that value explicitly.